src/main/resources/model.ttl

```diff
@@ -163,5 +169,29 @@ rm:user rdf:type owl:Class ;
 rm:impersonator rdf:type owl:Class ;
         rdfs:label "Impersonator"@en .
 
+###  http://onto.fel.cvut.cz/ontologies/record-manager/record-phase
+rm:record-phase rdf:type owl:Class ;
+        rdfs:subClassOf ufo:phase ;
+        rdfs:label "record state"@en .
+
+###  http://onto.fel.cvut.cz/ontologies/record-manager/open-record-phase
+rm:open-record-phase rdf:type owl:Class ;
+        rdfs:subClassOf rm:record-phase ;
+        rdfs:label "open record state"@en .
+
+###  http://onto.fel.cvut.cz/ontologies/record-manager/valid-record-phase
+rm:valid-record-phase rdf:type owl:Class ;
+        rdfs:subClassOf rm:record-phase ;
+        rdfs:label "valid record state"@en .
```

Comment:
I am confused now what will happen if i try to merge this ... I think it was already resolved somewhere.

Reply:
I can make a new pull request.